config/jobs/kubernetes/sig-release/cip/OWNERS

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- dims
+- hh
+- listx
+- tpepper
+reviewers:
```

Comment:
If these lists are the same (and they should be) can you just have the approvers section?